beta/src/content/apis/react/createFactory.md

```diff
@@ -2,19 +2,75 @@
 title: createFactory
 ---
 
-<Wip>
-
-This section is incomplete, please see the old docs for [createFactory.](https://reactjs.org/docs/react-api.html#createfactory)
-
-</Wip>
+<Intro>
 
+`createFactory` lets you create a function that creates a React element of a given `type`. The `type` argument can be either a tag name string (such as `div` or `span`), a React component type (a class or a function), or a React fragment type.
 
-<Intro>
+`createFactory` is typically invoked if you are using [React without JSX.](https://beta.reactjs.org/learn/add-react-to-a-website#run-the-jsx-preprocessor)
 
 ```js
 React.createFactory(type)
 ```
-
 </Intro>
 
+<Note>
+
+`createFactory` is considered legacy, and we encourage you to either use **JSX** or use `React.createElement()` directly instead.
+</Note>
+
 <InlineToc />
+
+## Usage {/*usage*/}
+
+### Creating React elements {/*creating-react-elements*/}
+
+In this example, we can render a React element of the `type` `button`.
+
+<Sandpack>
+
+``` js App.js
+import React from 'react';
+import { render } from 'react-dom';
+
+const MyButton = (reactElement = 'button') =>
+  (React.createFactory(reactElement))({
+    onClick: (evt) => {
+      evt.preventDefault();
+      alert("I was created by createFactory()");
+    }
+  }, 'Click');
```

Comment:
Can we replace `IIFE` with something similar to the below suggestion for better understanding/readability?

Also can we remove the `reactElement` parameter since the function itself was named as `MyButton` which says that it can be used to only create a button?
```suggestion
const MyButton = () => {
 const button = React.createFactory('button');
 const buttonProps = {
    onClick: (evt) => {
      evt.preventDefault();
      alert("I was created by createFactory()");
    }
  }
 return button(buttonProps, 'Click me!');
};
 ```